MatrixSDK/Room/State/MXRoomStateEventBuilder.swift

```diff
@@ -0,0 +1,77 @@
+// 
+// Copyright 2021 The Matrix.org Foundation C.I.C
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+// MARK: - Internal
+
+/// InitialStateEvent represents an intial state event
+struct InitialStateEvent {
+    let type: String
+    let stateKey: String = ""
+    let content: [String: Any]
+    
+    var jsonDictionary: [String: Any] {
+        return [
+            "type": type,
+            "stateKey": stateKey,
+            "content": content
+        ]
+    }
+}
+
+// MARK: - Public
+
+/// MXRoomStateEventBuilder enables to build initial state events
+@objcMembers
+public class MXRoomStateEventBuilder: NSObject {
```

Comment:
Can we call it `MXRoomInitialStateEventBuilder`?
It only creates such type of JSON, not JSON for full room state events.